conference/migrations/0014_auto_20190610_1352.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
It would be great to name the migration.

Reply:
Completely forgot - thanks for catching this :)